CodeGen/Generators/QuantityRelationsParser.cs

```diff
@@ -0,0 +1,174 @@
+﻿// Licensed under MIT No Attribution, see LICENSE file at the root.
+// Copyright 2013 Andreas Gullberg Larsen (andreas.larsen84@gmail.com). Maintained at https://github.com/angularsen/UnitsNet.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using CodeGen.JsonTypes;
+using Newtonsoft.Json;
+
+namespace CodeGen.Generators
+{
+    /// <summary>
+    ///     Parses the JSON file that defines the relationships (operators) between quantities
+    ///     and applies them to the parsed quantity objects.
+    /// </summary>
+    internal static class QuantityRelationsParser
+    {
+        /// <summary>
+        ///     Parse and apply relations to quantities.
+        ///     Each defined relation can be applied multiple times to one or two quantities depending on the operator and the operands.
+        ///
+        ///     The format of a relation definition is "Quantity.Unit operator Quantity.Unit = Quantity.Unit".
+        ///     "double" can be used as a unitless operand.
+        ///     "1" can be used as the left operand to define inverse relations.
+        /// </summary>
+        /// <param name="rootDir">Repository root directory.</param>
```

Comment:
A few examples with some actual quantities and units in a `<example></example>` tag could be helpful for the reader

Reply:
Done 👍